tox.ini

```diff
@@ -23,8 +23,10 @@ deps =
     pytest-runner
     compatibility: {[testenv:compatibility]deps}
 commands_pre =
-    # Remove the python-rsa backend
+    # Remove the python-rsa and python-ecdsa backends
     only: pip uninstall -y ecdsa rsa
+    # Remove just the python-rsa backend
+    nosra: pip uninstall -y rsa
```

Comment:
It says `nosra` here when it should be `norsa`.

Reply:
Doh. So it does. Thanks for the catch; looks like I also missed a few points where the pycrypto path is importing rsa.